db/messages_test.go

```diff
@@ -14,7 +14,7 @@ func MessagesTest(t *testing.T) {
 func MessagesSetup(t *testing.T) {
 	messages := QueryMessages()
 	assert.Equal(t, 8, len(messages))
-	assert.Equal(t, 52, messages[3].SSID)
+	assert.Equal(t, 52, int(messages[3].SSID))
```

Comment:
Don't think this is good to add - you would be chopping off the extra bits of a uint64 to an int. Why don't you convert 52 to `uint64` instead?

Reply:
Agreed. Will fix.